### ISSUE TITLE
Table option reorderable does not headers from beeing dragable and reoderable

### ISSUE THREAD

**Issue by s0x** [swimlane/ngx-datatable#413]
**I'm submitting a ...**
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
I disabled the reorderable option while using datatable-column directive and the corresponding template directives. I'm still able to drag and move the headers of the table. To disable this behavior I need to disable the draggable option for each column explicitly.

**Expected behavior**
Setting reorderable to false should disable the dragging and reorder behaviour in any case

* **Table version:** 4.0.0
* **Angular version:** 2.1.1

* **Browser:** tested with Chromium & Firefox. I suppose all

**Comment by amcdnl** [comment]
Odd, its setup to prevent that here: https://github.com/swimlane/ngx-datatable/blob/master/src/components/header/header.component.ts#L29

**Comment by s0x** [comment]
Indeed, I've seen that as well. I was not able to dig deeper into the issue yet though.
